src/interpolations.jl

```diff
@@ -1,47 +1,73 @@
+_ndofs(dofs::Tuple) = sum(idxs -> length(idxs), dofs)
+_nvertexdofs(ip) = _ndofs(Ferrite.vertexdof_indices(ip))
+_nedgedofs(ip) = _ndofs(Ferrite.edgedof_interior_indices(ip))
+_nfacedofs(ip) = _ndofs(Ferrite.facedof_interior_indices(ip))
+_ncelldofs(ip) = _ndofs(Ferrite.celldof_interior_indices(ip))
+
 # The constructors of `InterpolationInfo` require a `VectorizedInterpolation{InterfaceCellInterpolation}`
 # and not an `InterfaceCellInterpolation{VectorizedInterpolation,VectorizedInterpolation}`.
 # To create a `VectorizedInterpolation{InterfaceCellInterpolation}`, `InterfaceCellInterpolation` needs to be a `ScalarInterpolation`.
 
 """
-    InterfaceCellInterpolation(here::ScalarInterpolation, there::ScalarInterpolation) <: ScalarInterpolation
+    InterfaceCellInterpolation(ip::ScalarInterpolation) <: ScalarInterpolation
 
-An `InterfaceCellInterpolation` is an interpolation based on two interpolations on the faces of an `InterfaceCell`.
-If only one interpolation is given, it will be used for both faces.
+An `InterfaceCellInterpolation` is based on a regular interpolation which will be applied 
+to both faces of an `InterfaceCell`.
 
 # Fields
-- `here::ScalarInterpolation`: interpolation on the face "here"
-- `there::ScalarInterpolation`: interpolation on the face "there"
+- `base::ScalarInterpolation`: base interpolation
 """
-struct InterfaceCellInterpolation{shape, IPhere, IPthere} <: ScalarInterpolation{shape, Nothing}
-    here::IPhere
-    there::IPthere
-
-    function InterfaceCellInterpolation(here::IPhere, there::IPthere) where {IPhere<:ScalarInterpolation, IPthere<:ScalarInterpolation}
-        @assert getrefshape(here) == getrefshape(there) "For an `InterfaceCellInterpolation` the underlying interpolations need to be based on the same shape."
-        shape = get_interface_cell_shape(getrefshape(here))
-        return new{shape, IPhere, IPthere}(here, there)
+struct InterfaceCellInterpolation{shape, IP} <: ScalarInterpolation{shape, Nothing}
```

Comment:
I'd suggest to stick with the original parametrization of `ScalarInterpolation` here
```suggestion
struct InterfaceCellInterpolation{shape, order, IP} <: ScalarInterpolation{shape, order}
```

Reply:
I think that should allow you to remove the `Ferrite.getorder` definitions below

Reply:
Right, now that is well defined :+1: